interview-questions.md

```diff
@@ -7,60 +7,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is a software, a version control that can track and look at your codes like a time machine. It's created by Linux.
+  GitHub is a code hosting online platform for version control (git).
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: loose equality operator (==) and strict equality operator (===)
 
-  Researched answer:
+  Researched answer: Equality operators compare one value to the other and judge if the values equate to the same thing. If they do equate to the same a Boolean value of true is returned if not we get a Boolean value of false. There are two types of equality operators, the loose equality operator and the strict equality operator. The loose equality operator converts the two items to the same data type before making a comparison and the strict equality operator requires the two items being compared must be the same data type to return true.
```

Comment:
Yup, for sure. To expand on this, relational, logical, and equality operators return Boolean values.